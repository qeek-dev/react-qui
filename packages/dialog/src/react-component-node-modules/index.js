import 'normalize.css'

export { default as TextDialog } from './components/Dialog/TextDialog'
export { default as PopupDialog } from './components/Dialog/PopupDialog'
export { default as DialogContainer } from './components/DialogContainer/DialogContainer'
