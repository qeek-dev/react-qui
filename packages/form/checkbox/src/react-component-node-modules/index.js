export { default as Checkbox } from './components/Checkbox/Checkbox'
export { default as CheckboxGroup } from './components/CheckboxGroup/CheckboxGroup'

