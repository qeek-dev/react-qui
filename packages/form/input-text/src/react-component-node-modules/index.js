export { default as InputText } from './components/InputText/InputText';
