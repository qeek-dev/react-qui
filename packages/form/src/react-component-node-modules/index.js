export { default as Checkbox } from './components/Checkbox/Checkbox'
export {
  default as CheckboxGroup,
} from './components/CheckboxGroup/CheckboxGroup'
export { default as Label } from './components/Label/Label'
export { default as InputText } from './components/InputText/InputText'
export { default as Qselect } from './components/Qselect/Qselect'
