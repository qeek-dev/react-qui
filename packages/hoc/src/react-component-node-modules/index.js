export { default as withDraggable } from './hoc/withDraggable';
