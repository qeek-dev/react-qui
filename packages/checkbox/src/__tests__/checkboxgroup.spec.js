import React from 'react'
import ReactDOM from 'react-dom'
import { mount, shallow } from 'enzyme'
import { CheckboxGroup } from '../react-component-node-modules/index'
import { configure } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'

configure({ adapter: new Adapter() })

it('render <CheckboxGroup> component', () => {
  const checkboxGroup = shallow(
    <CheckboxGroup
      options={[
        { label: 'ADA', value: 'ada', checked: true },
        { label: 'NEO', value: 'neo', checked: true },
        { label: 'EOS', value: 'eos', checked: true },
      ]}
      span={30}
      theme="light"
      direction="row"
    />,
  )
  expect(checkboxGroup).toMatchSnapshot()
})

it('renders without crashing', () => {
  const div = document.createElement('div')
  ReactDOM.render(
    <CheckboxGroup
      options={[
        { label: 'ADA', value: 'ada', checked: true },
        { label: 'NEO', value: 'neo', checked: true },
        { label: 'EOS', value: 'eos', checked: true },
      ]}
      span={30}
      theme="light"
      direction="row"
    />,
    div,
  )
})

it('render CheckboxGroup correctly', () => {
  const checkboxGroup = mount(
    <CheckboxGroup
      options={[
        { label: 'ADA', value: 'ada', checked: true },
        { label: 'NEO', value: 'neo', checked: true },
        { label: 'EOS', value: 'eos', checked: true },
      ]}
      span={30}
      theme="light"
      direction="row"
    />,
  )
  expect(checkboxGroup.find('span').length).toEqual(3)
  expect(checkboxGroup.children().text()).toEqual('ADANEOEOS')
})
