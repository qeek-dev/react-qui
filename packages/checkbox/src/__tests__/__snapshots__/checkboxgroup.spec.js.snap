// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render <CheckboxGroup> component 1`] = `
<styled.div>
  <withStateHandlers(Checkbox)
    checked={true}
    direction="row"
    key="ada"
    onChange={[Function]}
    span={30}
    theme="light"
    value="ada"
  >
    ADA
  </withStateHandlers(Checkbox)>
  <withStateHandlers(Checkbox)
    checked={true}
    direction="row"
    key="neo"
    onChange={[Function]}
    span={30}
    theme="light"
    value="neo"
  >
    NEO
  </withStateHandlers(Checkbox)>
  <withStateHandlers(Checkbox)
    checked={true}
    direction="row"
    key="eos"
    onChange={[Function]}
    span={30}
    theme="light"
    value="eos"
  >
    EOS
  </withStateHandlers(Checkbox)>
</styled.div>
`;
